app/src/org/commcare/fragments/connect/ConnectDownloadingFragment.java

```diff
@@ -30,22 +46,112 @@ public static ConnectDownloadingFragment newInstance() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        startAppDownload();
+    }
+
+    private void startAppDownload() {
+        ResourceInstallUtils.startAppInstallAsync(false, 11, (CommCareTaskConnector)getActivity(), TEST_APP_LINK);
```

Comment:
It is just a constant to recognise a particular task in case there are multiple tasks running simultaneously. In this case it doesn't matter what value we assign to it since there would only be one task at a time on this page. https://github.com/dimagi/commcare-android/pull/2706/commits/7cc2037c56aa822fc182018729e941bde5fb72cc